pages/common/nim.md

```diff
@@ -0,0 +1,24 @@
+# nim
+
+> The Nim compiler.
+> Processes, compiles and links Nim language source files.
+
+- Compile a source file:
+
+`nim compile {{file}}`
```

Comment:
Maybe `{{path/to/file.ext}}`, where `.ext` is the default extension for a nim source file? Also on the other examples below.

Reply:
The `.nim` extension can be ommitted, e.g. a file called `main.nim` can be compiled using `nim compile main`.

Reply:
But can a file called main.txt be compiled using `nim compile main` ? If not, then I think we should add the extension.

Reply:
@agnivade No, it only detects the `.nim` extension; I think it makes sense to add the extension in the example.